docs/HTTP-batchsink.md

```diff
@@ -14,6 +14,22 @@ Properties
 
 **batchSize:** Batch size. Defaults to 1. (Macro enabled)
 
+**Write JSON As Array:** Whether to write the JSON as an array. Defaults to false. Ignored if batch size is 1.
```

Comment:
It's simpler to have this always apply instead of having it depend on the batch size value. 

It also allows somebody to send arrays of 1 element if they want to for some reason.

Reply:
Updated to let user use size 1 array